app/views/admin/deploy_groups/_fields.html.erb

```diff
@@ -13,6 +13,15 @@
     </div>
   </div>
 
+  <% if ENV["SECRET_STORAGE_BACKEND"] == "SecretStorage::HashicorpVault" %>
```

Comment:
I think we do a similar check somewhere else ...
something like `SecretStorage.backend == SecretStorage::HashicorpVault` ... or checking against a constant ...